Makefile

```
@@ -54,12 +54,10 @@ run:
 	docker-compose -p edgex -f docker-compose${NO_SECURITY}${UI}${ARM64}.yml up -d ${SERVICES}
 
 down:
-	docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down
+	docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down ${CLEAN}
 
-clean: down
-	-docker rm $$(docker ps --filter "network=edgex_edgex-network" --filter "network=edgex_default" -aq) 2> /dev/null
-	docker volume prune -f && \
-	docker network prune -f
+clean:
+	CLEAN=-v make down
```

This actually took me a bit to figure out what you were doing here, because i thought you were just assigning the `CLEAN` variable to that whole value, until I realized you were using a sub-make. In that case, I think this is the preferred way:

```suggestion
	$(MAKE) CLEAN=-v down
```

It is recommended to call sub-makes using `$(MAKE)`, and `CLEAN` is a specific variable to your make command, so I would put it after the make instead of before like env vars (even if it works both ways).

---

In cases like this, I would typically implement it as a function with arguments as opposed to sub-makes. That could just be my personal preference however:
```make
COMPOSE_DOWN = docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down $1

down:
	$(call COMPOSE_DOWN,)
	# or simply use $(COMPOSE_DOWN) if you do not care that $1 is undefined

clean:
	$(call COMPOSE_DOWN,-v)
```

---

Nice, Thanks!